### PR TITLE
Add MQTT support

### PR DESCRIPTION
**Pull request by DoomHammer**

### CODE REVIEW

**Review by reinhrst — COMMENTED** [review]
Note that the Dockerfile is not actually being used at the moment (it was for balena). Still it's a good way to make sure that we document somewhere how to  install the thing (maybe it would have been better to use a requirements.txt, and then import that into the dockerfile, but that's another thing

File: count.py [hunk 1]
```diff
@@ -9,11 +9,46 @@
...
 import cv2
 from gpiozero import LEDCharDisplay
+from paho.mqtt import client as mqtt_client
+
 
 counts = None
 newimage = None
 
 
+broker = '100.89.232.16'
```
Comment (reinhrst) [inline count.py]: Any chance we want to get this, and the port, as env vars and then set them in docker (although, iirc, voncount currently runs in `supervised`, not docker)

**Review by reinhrst — APPROVED** [review]
Looks good, some small comments. Happy to merge, however cleanest is to only merge it when we're deploying it, right :)

File: count.py (hunk 1 above)
Comment (DoomHammer) [reply count.py]: using env var would definitely be a much better idea

**Pull request merged**